tests/test_anon.py

```diff
@@ -0,0 +1,53 @@
+from pydicom import dcmread
+from pydicom.data import get_testdata_file
+from dicomanonymizer import dicomfields
+from dicomanonymizer import anonymize_dataset
+
+
+def test_pydicom_data():
```

Comment:
The test look good, but I think it would be best to split it into several tests so when something goes wrong, you known immediately which test failed instead of having to go through the test log file.

You could also use a pytest fixture to generate the original dataset then re-use it for each test.

```python
@pytest.fixture(scope="module")
def CT_small_anonymized_dataset():
  orig_ds = dcmread(sample_data, read=True)
  return  (orig_ds, anonymize_dataset(orig_ds))

def test_deleted_tags_are_removed(CT_small_anonymized_dataset):
  orig_ds, anon_ds = CT_small_anonymized_dataset
  deleted_tags = dicomfields.X_TAGS
  for tt in deleted_tags:
    if len(tt) == 2 and tt in orig_ds:
      assert tt not in anon_ds

def test_changed_tags_are_replaced(CT_small_anonymized_dataset):
  orig_ds, anon_ds = CT_small_anonymized_dataset
  
...
```
The `scope="module"` means that the fixture is going to be executed once and its output will be shared to all tests using it. (https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session)

Reply:
I had to make a small adjustment to `CT_small_anonymized_dataset()`, but this worked. Thank you for the suggestions.